service-monitor/src/main/java/com/yahoo/vespa/service/duper/ConfigServerApplication.java

```diff
@@ -1,15 +1,29 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.vespa.service.duper;
 
+import com.yahoo.component.Version;
 import com.yahoo.config.provision.ClusterSpec;
 import com.yahoo.config.provision.NodeType;
 import com.yahoo.vespa.applicationmodel.ServiceType;
 
 /**
  * A service/application model of the config server with health status.
+ *
+ * @author hakonhall
  */
 public class ConfigServerApplication extends ConfigServerLikeApplication {
+
     public ConfigServerApplication() {
         super("zone-config-servers", NodeType.config, ClusterSpec.Type.admin, ServiceType.CONFIG_SERVER);
     }
+
+    @Override
+    public ClusterSpec getClusterSpecWithVersion(Version version) {
```

Comment:
Consider moving this method up into ConfigServerLikeApplication, which allows removing the duplicate method in ControllerApplication.